examples/offline_chat_with_tools.py

```diff
@@ -0,0 +1,138 @@
+# ruff: noqa
+import json
+import random
+import string
+
+from vllm import LLM
+from vllm.sampling_params import SamplingParams
+
+# This script is an offline demo for running Pixtral.
```

Comment:
Note that this is not a demo for Pixtral

Reply:
Nice catch!